### PR TITLE
Volvo pr day 13 and 14

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 1]
```diff
@@ -689,18 +689,18 @@ SHIKI slashed you across the chest, and you were on the verge of death.
...
 -- そこに父が駆けつけて、シキを止めたんです」
-That's when father rushed in and stopped him."
+That's when Father rushed in and stopped him."
 }
 [sha:f8141ac87f5af1ac7bb922e34240f27cf93740ce]{
 -- Page 46, Offset 31801.
 -- 「……理性を失っていたシキを止めるには、息の根を止めるしかない。遠野家の当主には、そうやって反転してしまった一族の者を処理する義務があります。
 "...Once SHIKI had lost all reason, the only way to stop him was to kill him.
- The head of the Tohno family has the duty of disposing of inverted family members.
+ The head of the Tohno family has the duty of disposing inverted family members.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_1.txt:698]: `of` is correct here

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 2]
```diff
@@ -799,7 +799,7 @@ Nii-san, do you really not find any of this hard to believe...?"
 -- Page 54, Offset 31821.
 -- 「あのな。おまえがこんな嘘をつくワケがないし、その、こういう話には慣れているというか。
 "Come on. There's no way you'd make something like this up,
- and, well, I guess I'm sorta used to hearing things like this by now.
+ and well, I guess I'm sorta used to hearing things like this by now.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_1.txt:802]: I'd include that comma, it's how I'd insert pauses when reading this

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 3]
```diff
@@ -531,6 +531,7 @@ Roa is a reincarnating vampire. It's only natural that his appearance, character
 [sha:2a81be82906961c56d3b4f57182dde176aef28be]{
 -- Page 119, Offset 32004.
 -- 「いいですか遠野くん。ロアが転生先に選んだ肉体は、ロアとは別の人格・人間として育ちます。知性が十分に育たなければロアは己を発揮できませんから。
+// why is themselves used here when it's referring to Roa himself?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:534]: Mistake I guess

File: script/Arcueid/Day 13/13_00_ARC13_4.txt [hunk 4]
```diff
@@ -572,7 +572,7 @@ Can Roa not see this?
 [sha:51448463ffbab879ddb7006429f26252a4791698]{
 -- Page 338, Offset 32604.
 -- あの、“死”すら立ち入れない、膨大な力のうねりが。
-This billowing wave of energy so potent that death itself cannot trespass upon it.
+This billowing wave of energy is so potent that death itself cannot trespass upon it.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_4.txt:575]: no `is` here

File: script/Arcueid/Day 13/13_00_ARC13_4.txt [hunk 5]
```diff
@@ -847,7 +847,7 @@ With each step they take, they begin to regain their shape.
 -- Page 355, Offset 32653.
 -- このように―――術者が滅びようと発動する自動詠唱は、近年において完成の域をみた”」
 Advancements such as this one―――
-Automatic Chanting, triggered just as the user is about to perish, is something that's been perfected in recent years."
+%{i}Automatic Chanting%{/i}, triggered just as the user is about to perish, is something that's been perfected in recent years."
```
Comment (yk1512) [inline script/Arcueid/Day 13/13_00_ARC13_4.txt:850]: replace chanting with incantation (lore correction)

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 6]
```diff
@@ -689,18 +689,18 @@ SHIKI slashed you across the chest, and you were on the verge of death.
...
 -- そこに父が駆けつけて、シキを止めたんです」
-That's when father rushed in and stopped him."
+That's when Father rushed in and stopped him."
 }
 [sha:f8141ac87f5af1ac7bb922e34240f27cf93740ce]{
 -- Page 46, Offset 31801.
 -- 「……理性を失っていたシキを止めるには、息の根を止めるしかない。遠野家の当主には、そうやって反転してしまった一族の者を処理する義務があります。
 "...Once SHIKI had lost all reason, the only way to stop him was to kill him.
- The head of the Tohno family has the duty of disposing of inverted family members.
+ The head of the Tohno family has the duty of disposing inverted of family members.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Arcueid/Day 13/13_00_ARC13_1.txt:698]: you didnt see that

**Review by rschlaikjer — APPROVED** [review]
Approved pending final correction

**Pull request merged**